### PR TITLE
Cleanup dependencies

### LINKED ISSUE #9: Depends on "Plone" for no reason
This package depends on "Plone" for no reason and pulls in requirements which might not be wanted.

### PR DESCRIPTION
**Pull request by thet**
Declare the direct dependencies of this project and remove the dependency "Plone" which might pull in unwanted other dependencies.

Fixes: #9

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,11 +51,15 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "Plone",
```
Comment (gforcada) [inline setup.py:54]: there might be some hidden functionality coming from Plone (or any of its dependencies 😓 ) as tests do fail 😕

Comment (ericof) [reply setup.py:54]: Maybe adding Products.CMFPlone, and plone.app.testing should fix this?

Comment (thet) [reply setup.py:54]: will try!

**Comment by ericof** [comment]
Closing it in favor of #14

**Pull request closed without merging**